shared/blob_utils.py

```
@@ -1,50 +1,138 @@
+    """
+    async with BlobServiceClient.from_connection_string(BLOB_CONN_STR) as service:
+        # Check/create container
+        try:
+            container_client: ContainerClient = service.get_container_client(BLOB_CONTAINER)
+            await container_client.get_container_properties()
+        except ResourceNotFoundError:
+            print(f"Container '{BLOB_CONTAINER}' not found. Creating it.")
+            try:
+                await service.create_container(BLOB_CONTAINER)
+            except ResourceExistsError:
+                pass  # Possible race condition where it was created just after check
```

Ruff (0.11.9)

Use `contextlib.suppress(ResourceExistsError)` instead of `try`-`except`-`pass`

(SIM105)